lib/task-data/base-tasks/generate-tag.js

```diff
@@ -0,0 +1,13 @@
+// Copyright 2016, EMC, Inc.
+
+'use strict';
+
+module.exports = {
+    friendlyName: 'Generate Tag',
+    injectableName: 'Task.Base.Catalog.GenerateTag',
```

Comment:
@zyoung51 @yyscamper many of our injector names (especially older ones) have no strong reasoning behind them, and may not actually make sense anymore :). I think we initially nested it under `Catalog` because the job relied upon and searched through the catalogs.